opencv_apps/package.xml

```diff
@@ -27,6 +27,15 @@
   <run_depend>roscpp</run_depend>
   <run_depend>std_srvs</run_depend>
 
+  <test_depend>rostest</test_depend>
+  <test_depend>rosbag</test_depend>
+  <test_depend>rosservice</test_depend>
+  <test_depend>rostopic</test_depend>
+  <test_depend>procps</test_depend>
+  <test_depend>image_proc</test_depend>
+  <test_depend>image_view</test_depend>
```

Comment:
This line introduces a circular dependency on the repository level between this repo (B) and  `image_pipeline` (A) . E.g. the package `camera_calibration`in A depends on `cv_bridge` in B and `opencv_apps` in B depends on `image_view` in A.

As a result none of the two repos can be released into a new distro without ignoring packages.